structure/0062-quilt-community-collab.md

```diff
@@ -0,0 +1,250 @@
+
+## Process: Removing a community
+
+While we hope that it won't ever be necessary, it's important that we have the ability to remove communities from the program when it becomes necessary.
+
+### Starting the process
+
+This process is largely Manager-driven. The Manager team may decide to start this process due to concerns shared with them (by anyone) or events witnessed by them directly --- included, but not limited to:
+
+* Substantiated concerns regarding a community's staff teams perpetuating harmful behaviour --- such as paedophilia, physical violence, or ableism, racism, transphobia or other bigotry
+* Sudden changes in community leadership with no appropriate reasoning, or a problematic agenda
+* Changes in community rules or purpose, such that the community no longer fits into the program
+
+This is not an exhaustive list, but the related discussions that happen as part of this process are free to debate the reasoning for the call for removal if needed. However, the removal process is unlikely to be an appropriate resolution for issues that may be easily addressed through communication with the community in question.
+
+### Removal process
+
+The removal process is defined as follows:
+
+1. If required, the Manager team may replace the roles currently granted to the representatives from the community in question with the Invitee role --- This must be communicated to those representatives, but the Managers are not required to communicate the reason if they believe it's not the best course of action
+2. The Manager team must post a message visible to the rest of the Program's members, clearly explaining the concerns and the reasoning behind starting the removal process, and with an associated discussion thread --- this message and thread must not be visible to the representatives of the community in question
+3. Once the voting-eligible Program members have informally agreed that everyone's had enough time to read the message and ask questions, the vote may be created (with options to approve or deny the vote) --- it must follow the voting process outlined earlier in this document, but **it is not eligible to be vetoed by Program members**
+4. Once the vote has concluded, the discussion channel must be archived and kept read-only, so it can be reviewed later on if needed
+
+### Interpreting the vote
+
+**The vote has passed** when more than 50% of the votes cast at the time of closing are positive. The community is removed from the Program and any listings therein.
+
+Otherwise, **the vote has failed.** Internal discussions can be removed as needed. The community is reintegrated, and any roles that were removed are returned.
+
+## Process: Removing an Individual Collaborator
+
+While we hope that it won't ever be necessary, it's important that we have the ability to remove Individual Collaborators from the program when it becomes necessary.
+
+This process matches the process for removing a community.
```

Comment:
```suggestion
The process for removing an Individual Collaborator matches the process for removing a community.
```
I don't see the need for a heading and an introduction that's longer than the actual content of the heading, while also basically repeating what was just said.